it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/SessionTestStage.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.it;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import com.expediagroup.streamplatform.streamregistry.graphql.client.CreateSessionMutation;
+import com.expediagroup.streamplatform.streamregistry.graphql.client.RenewSessionMutation;
+import com.expediagroup.streamplatform.streamregistry.it.helpers.AbstractTestStage;
+
+public class SessionTestStage extends AbstractTestStage {
+
+  @Override
+  public void create() {
+    setFactorySuffix("create");
+
+    CreateSessionMutation.Create create = createSession();
+
+    assertNotNull(create);
+  }
+
+  @Override
+  public void update() {
+
```

Comment:
Are these going to be implemented?

Reply:
No, but the parent class has some utility functions that are used and these tests are abstract in parent class, what would you advise?